drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -519,14 +519,17 @@ function DragAndDropTemplates(configuration) {
         var is_item_placed = function(i) { return i.is_placed; };
         var items_placed = $.grep(ctx.items, is_item_placed);
         var items_in_bank = $.grep(ctx.items, is_item_placed, true);
-        var item_bank_properties = {};
+        var item_bank_properties = {
+            attributes: {
+                'role': 'group',
+                'aria-label': gettext('Item Bank')
+            }
+        };
         if (ctx.item_bank_focusable) {
-            item_bank_properties.attributes = {
-                'tabindex': 0,
-                'dropzone': 'move',
-                'aria-dropeffect': 'move',
-                'role': 'button'
-            };
+            item_bank_properties.attributes['tabindex'] = 0;
+            item_bank_properties.attributes['dropzone'] = 'move';
+            item_bank_properties.attributes['aria-dropeffect'] = 'move';
+            item_bank_properties.attributes['role'] = 'button';
```

Comment:
@mtyaka I wonder if it has some special meaning to screen readers. You have certainly [added it](https://github.com/edx-solutions/xblock-drag-and-drop-v2/commit/0e07b79936020a862f06ca662cb83e34de8cfa96#diff-cef30416b48c3e5d4dc05c6922a2729bR296) for a reason - do you remember what was that?